src/server/ideController.py

```diff
@@ -47,7 +47,30 @@ def create_tree_nodes_dict(nodes):
 
 def set_author_bool_in_dict(serialized_changes, user):
   for sc in serialized_changes:
-    sc['is_from_you'] = (sc['author'] == user)
+    sc['author'] = (sc['author'] == user)
```

Comment:
The first occurence of 'author' key was changed to 'is_from_you', else the author name will be lost for further itteration in the user send loop.
(Ex: gives "if False == grondinjc" ...)
The correct line should be the original:
`sc['is_from_you'] = (sc['author'] == user)`